ci.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/sh
+set -exo
+set -u pipefail
+
+
+if CI=true; then
+  curl https://storage.googleapis.com/pub/gsutil.tar.gz | tar xfz - \
```

Comment:
Can you use newlines here?
It's harder to read with the semcolon prefix and line continuations.
I don't think they are necessary.

Reply:
Agreed. Please see: https://github.com/cncf/apisnoop/pull/144/commits/82e732a24cecfba00369f54c753594f9ae447638